manifests/2.1.0/opensearch-dashboards-2.1.0.yml

```diff
@@ -10,6 +10,11 @@ components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
     ref: '2.1'
+    checks:
+      - npm:package:version
```

Comment:
Is this something new? Don't see it in 2.0.1 [manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.0.1/opensearch-dashboards-2.0.1.yml#L10-L12)

Reply:
I'm not particularly sure but it was on the 2.0 manifest

Reply:
Okay looks like for 2.0.1 it was missed.